packages/firebase_data_connect/firebase_data_connect/example/android/app/google-services.json

```diff
@@ -0,0 +1,154 @@
+{
+  "project_info": {
+    "project_number": "958807142248",
+    "project_id": "movie-app-dart",
```

Comment:
Should we get access to this project to better maintain this example app if needed?

Reply:
Replaced with e2e project.